text/label/sentencepiece/apply.py

```diff
@@ -0,0 +1,91 @@
+import logging
+import shutil
+import subprocess as sp
+import tempfile
+import os
+
+from sisyphus import Job, Task, tk, gs
+
+import i6_core.util as util
+
+try:
+    import sentencepiece
+except ImportError:
+    if not hasattr(gs, "WARNING_NO_SENTENCEPIECE") or gs.WARNING_NO_SENTENCEPIECE is True:
+        logging.warning(
+            "The package 'sentencepiece' is not installed in the manager python env. Please make sure it is installed "
+            "in the python environment running the Sisyphus worker. To suppress this warning set "
+            "'WARNING_NO_SENTENCEPIECE=False' in the settings.py"
+        )
+
+
+class ApplySentencepieceToTextJob(Job):
+    """
+    Apply sentencepiece model on a text file, basically a wrapper for spm.encode
+    """
+
+    def __init__(
+        self,
+        *,
+        text_file: tk.Path,
+        sentencepiece_model: tk.Path,
+        map_unk: bool = False,
+        gzip_output: bool = True,
+        mini_task: bool = True,
```

Comment:
Is this a good default here? I think this might make more sense:
```suggestion
        mini_task: bool = False,
```